src/components/seconardynavnew/mobile-styles.ts

```diff
@@ -1,11 +1,23 @@
 import { ThemeUIStyleObject } from 'theme-ui';
 
+export const navWrapperStyles: ThemeUIStyleObject = {
+    boxSizing: 'border-box',
+    background: '#fff',
+    backgroundColor: 'white',
+    display: ['block', 'block', 'block', 'none'],
```

Comment:
`display: ['block', null, null, 'none']` (unless im misunderstanding that on my end)

Reply:
ah yep good catch!